src/subscription_manager/managercli.py

```diff
@@ -1542,6 +1542,11 @@ def _determine_owner_key(self, cp):
         if len(owners) == 1:
             return owners[0]['key']
 
+        if len(owners) > 1:
+            org_keys = [owner['key'] for owner in owners]
+            print(_('Hint: User "%s" is member of following organizations: %s') %
+                  (self.username, ', '.join(org_keys)))
```

Comment:
Maybe, we can also display here `owner['displayName']` as we do in `subscription-manager owners` command.